website/docs/d/alert_configurations.html.markdown

```diff
@@ -0,0 +1,71 @@
+---
+layout: "mongodbatlas"
+page_title: "MongoDB Atlas: Alert Configurations"
+sidebar_current: "docs-mongodbatlas-datasource-alert-configurations"
+description: |-
+    Describe all Alert Configurations in Project.
+---
+
+# Data Source: mongodbatlas_alert_configurations
```

Comment:
can you also add an example on this file so users know how to use new data source?

Reply:
Yep! Added an example on how to use the new output fields